lib/src/components/NxAccordion/__tests__/NxAccordion.test.tsx

```diff
@@ -5,277 +5,221 @@
+    const { rerender, container } = quickRender();
+
+    rerender(
+      <NxAccordion>
+        <NxAccordion.Header>
+          <span className='foo'>Foo</span>
+        </NxAccordion.Header>
+        <span className='bar'>Bar</span>
+      </NxAccordion>
+    );
+
+    expect(container.querySelector('.nx-accordion__header .foo')).toBeInTheDocument();
+    expect(container.querySelector('.nx-accordion__content .bar')).toBeInTheDocument();
   });
 
   describe('Header', function() {
     it('renders the NxAccordion.Header as a <summary> containing a nx-accordion__summary-wrapper', function() {
-      const component = getMountedComponent({
-            children: (
-              <NxAccordion.Header id="headerId">
-                <span>Foo</span>
-              </NxAccordion.Header>
-            )
-          }),
-          header = component.find('summary'),
-          wrapper = header.children();
-
-      expect(header).toExist();
-      expect(header).toMatchSelector('#headerId');
-      expect(wrapper).toHaveClassName('nx-accordion__summary-wrapper');
-      expect(wrapper).toContainReact(<span>Foo</span>);
-    });
+      const { rerender, container } = quickRender();
 
-    it('sets the nx-accordion__header class along with any provided className on the header', function() {
-      const component = getMountedComponent({
-            className: 'accordion-class',
-            children: (
-              <NxAccordion.Header className="header-class">
-                <span>Foo</span>
-              </NxAccordion.Header>
-            )
-          }),
-          header = component.find('summary');
-
-      expect(header).toHaveClassName('nx-accordion__header');
-      expect(header).toHaveClassName('header-class');
-      expect(getMountedComponent({ children: <NxAccordion.Header /> }).find('summary'))
-          .toHaveClassName('nx-accordion__header');
-    });
+      rerender(
+        <NxAccordion.Header id="headerId">
+          <span>Foo</span>
+        </NxAccordion.Header>
+      );
 
-    it('renders an icon with nx-accordion__chevron class as the first child of the header wrapper', function() {
-      const component = getMountedComponent({
-            children: (
-              <NxAccordion.Header>
-                <span>Foo</span>
-                <div className="nx-btn-bar">
-                  <NxButton />
-                </div>
-              </NxAccordion.Header>
-            )
-          }),
-          wrapper = component.find('summary').children();
-
-      expect(wrapper.children().first()).toMatchSelector(NxFontAwesomeIcon);
-      expect(wrapper.children().at(1)).toMatchSelector('span');
-      expect(wrapper.children().last()).toMatchSelector('div');
-
-      expect(wrapper.find(NxFontAwesomeIcon)).toHaveClassName('nx-accordion__chevron');
+      expect(container.querySelector('#headerId')).toBeInTheDocument();
+      expect(container.querySelector('.nx-accordion__summary-wrapper')).toBeInTheDocument();
```

Comment:
This class is an impl detail

Reply:
> This class is an impl detail

Removed here: 7907f12daefbcc73441d1fc90f29256059f91c55

Reply:
> Removed here: https://github.com/sonatype/sonatype-react-shared-components/commit/7907f12daefbcc73441d1fc90f29256059f91c55

The test name indicates that it checks whether the header renders a `<summary>`.  I don't see that happening.  In addition, the test name still talks about `.nx-accordion__summary-wrapper`.  That part should be removed.

Reply:
> > Removed here: [7907f12](https://github.com/sonatype/sonatype-react-shared-components/commit/7907f12daefbcc73441d1fc90f29256059f91c55)
> 
> The test name indicates that it checks whether the header renders a `<summary>`. I don't see that happening. In addition, the test name still talks about `.nx-accordion__summary-wrapper`. That part should be removed.

Added here: 735be7af5df2ea43ca7885546fb8c2a3593882e0